README.md

```diff
@@ -385,6 +386,53 @@ config := hazelcast.NewConfig()
 config.NetworkConfig().AddAddress("some-ip-address:port")
 hazelcast.NewClientWithConfig(config)
 ```
+
+### 1.4.3. Hazelcast Properties
+
+Hazelcast property is a property which can be set programmatically through `config.SetProperty` or with an environment variable.
```

Comment:
While configuring your Go client, you can use various system properties provided by Hazelcast to tune its clients. These properties can be set programmatically through `config.SetProperty` or by using an environment variable.